pebble/src/test/java/com/mitchellbosecke/pebble/extension/ExtensionCustomizerTest.java

```diff
@@ -37,15 +39,16 @@ void upperFilterCannotBeUsed() throws IOException {
             () -> "Expect upper-Filter to not exist, actual Problem: " + exception.getMessage());
   }
 
-  private static class RemoveUpper extends ExtensionCustomizer {
+  private static class RemoveUpperCustomizer extends ExtensionCustomizer {
 
-    public RemoveUpper(Extension core) {
+    public RemoveUpperCustomizer(Extension core) {
       super(core);
     }
 
     @Override
     public Map<String, Filter> getFilters() {
-      Map<String, Filter> filters = new HashMap<>(super.getFilters());
+      Map<String, Filter> filters = Optional.ofNullable(super.getFilters()).map(HashMap::new)
```

Comment:
just curious: is there a reason why null is acceptable over an empty list if filters and others are not provided?